newIDE/app/src/MainFrame/index.js

```diff
@@ -1602,6 +1603,14 @@ class MainFrame extends React.Component<Props, State> {
     editorTab.editorRef.updateToolbar();
   }
 
+  forceUpdateObjectsList() {
+    const editorTab = getCurrentTab(this.state.editorTabs);
+    if (
+      editorTab.key.startsWith('layout ' + editorTab.name)
```

Comment:
This is risky as a change in this key later could silently break this. Can you think of another way of doing this?

Reply:
I've was an idea in a previous commit, but not used.
Add a props type for earch type of tab editor.
This is an idea, maybe another way existing, for get the type without this addition.
https://github.com/4ian/GDevelop/pull/1429/commits/27331594e878eed7aae543128286888e86d53c2a

Reply:
Well yes, it had no provision to know the type of the editor, hence had to use this work-around as I think key uniquely determines each type of editor! :) 

Thanks @Bouh for the idea, I'll try and implement it.